doc/54_Git.md

```diff
@@ -1,7 +1,7 @@
 # Git Integration
 
-Theia ships with a couple of Git-related features. Please note that these are all meant to be
-convenient UIs on top of the command line `git`, which is available in the terminal as well.
+Theia integrates closely with Git. All of those features are convenient UIs on top of the
```

Comment:
Theia integrates closely with Git _by offering_ convenient UIs

Reply:
good tip, I fill fix my fix in the follow-up